lib/pure/times.nim

```diff
@@ -174,92 +140,348 @@ type
     months*: int      ## The number of months
     years*: int       ## The number of years
 
-{.deprecated: [TMonth: Month, TWeekDay: WeekDay, TTime: Time,
-    TTimeInterval: TimeInterval, TTimeInfo: TimeInfo].}
-
-proc getTime*(): Time {.tags: [TimeEffect], benign.}
-  ## gets the current calendar time as a UNIX epoch value (number of seconds
-  ## elapsed since 1970) with integer precission. Use epochTime for higher
-  ## resolution.
-proc getLocalTime*(t: Time): TimeInfo {.tags: [TimeEffect], raises: [], benign.}
-  ## converts the calendar time `t` to broken-time representation,
-  ## expressed relative to the user's specified time zone.
-proc getGMTime*(t: Time): TimeInfo {.tags: [TimeEffect], raises: [], benign.}
-  ## converts the calendar time `t` to broken-down time representation,
-  ## expressed in Coordinated Universal Time (UTC).
-
-proc timeInfoToTime*(timeInfo: TimeInfo): Time
-    {.tags: [TimeEffect], benign, deprecated.}
-  ## converts a broken-down time structure to
-  ## calendar time representation. The function ignores the specified
-  ## contents of the structure members `weekday` and `yearday` and recomputes
-  ## them from the other information in the broken-down time structure.
-  ##
-  ## **Warning:** This procedure is deprecated since version 0.14.0.
-  ## Use ``toTime`` instead.
+  Timezone* = object ## Timezone interface for supporting ``DateTime``'s of arbritary timezones.
```

Comment:
Very nitpicky: ``TimeZone``

Reply:
I looked it up before I went with `Timezone`, and it seems both forms are acceptable: https://english.stackexchange.com/questions/3934/time-zone-vs-timezone. I subjectively prefer `Timezone`, because in the context of the `times` module it doesn't really makes sense to think of a `Timezone` as `Time` + `Zone`, since `Time` means something very specific in the module.